logging/slf4j/src/main/java/io/helidon/logging/slf4j/Slf4jMdcProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.logging.slf4j;
+
+
+import io.helidon.logging.spi.MdcProvider;
+
+import org.slf4j.MDC;
+
+/**
+ * Provider for setting MDC values to the Slf4j MDC support.
```

Comment:
Javadoc should mark it as a service loader service, so people do not try to use it directly